packages/ibm-products/src/components/UserAvatar/UserAvatar.js

```diff
@@ -46,43 +46,51 @@ const componentName = 'UserAvatar';
  * TODO: A description of the component.
  */
 
-const defaults = {
-  renderIcon: () => <User size={32} />,
-  tooltipAlignment: 'bottom',
-  tooltipText: 'Thomas J. Watson',
-};
-
 export let UserAvatar = React.forwardRef(
   (
     {
       // The component props, in alphabetical order (for consistency).
       backgroundColor,
       className,
+      name,
       /* TODO: add other props for UserAvatar, with default values if needed */
-      renderIcon = defaults.renderIcon,
-      tooltipText = defaults.tooltipText,
-      tooltipAlignment = defaults.tooltipAlignment,
+      renderIcon,
```

Comment:
since the user isn't passing props directly to the icon this way we can reference this as a component to be consumed directly. i'll follow this up in an additional comment `renderIcon: RenderIcon`